xmtp_mls/src/groups/sync.rs

```diff
@@ -515,7 +532,20 @@ impl MlsGroup {
                     openmls_group,
                 )
                 .await?;
-                openmls_group.merge_staged_commit(provider, sc)?;
```

Comment:
This change will no longer return early on error

Reply:
good catch, im checking all the other logs I added. This is fixed here c8db82024e12b1f43f85d3ec61c960eb8654c04a